GUI/SettingsDialog.cs

```diff
@@ -399,8 +399,16 @@ private void CheckForUpdatesButton_Click(object sender, EventArgs e)
 
         private void InstallUpdateButton_Click(object sender, EventArgs e)
         {
-            Hide();
-            Main.Instance.UpdateCKAN();
+            if (AutoUpdate.CanUpdate)
+            {
+                Hide();
+                Main.Instance.UpdateCKAN();
+            }
+            else
+            {
+                GUI.user.RaiseError("Can't autoupdate. Please check https://github.com/KSP-CKAN/CKAN/ for help!");
```

Comment:
Would someone opening this URL be able to figure out what was going on and/or what to do about it?

Reply:
Ooh, that's a good point. Just merged this, unfortunately. Pointing to the latest release would probably be better, no?

Reply:
Oups, you were a bit faster than me.
Actually, I wanted to ask if this would work first, before detailing the error message...
Well, should I open a new PR?

Reply:
Make it point to https://github.com/KSP-CKAN/CKAN/releases/latest I think. @HebaruSan , does that seem like a useful link to you?

Reply:
Yes, maybe something like:
"Error during update.
Can't update automatically, because ckan.exe is read-only or we are not allowed to overwrite it. Please update manually via https://github.com/KSP-CKAN/CKAN/releases/latest ." ?

Reply:
That message sounds good to me!